documentation.html

```diff
@@ -54,21 +54,21 @@ <h2>this is a warning alert</h2>
                         <h2>this is a info alert</h2>
                         <a href="">&times;</a>
                     </div>
-                </div><iframe
+                </div>
+                <iframe
                 src="https://carbon.now.sh/embed?bg=rgba%2849%2C106%2C181%2C1%29&t=seti&wt=none&l=htmlmixed&width=408&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=36px&ph=39px&ln=false&fl=1&fm=Fira+Code&fs=13px&lh=156%25&si=false&es=2x&wm=false&code=%253Cimg%2520class%253D%2522circular-x-large-avatar%2522%2520src%253D%2522.%252Fassets%252Fprofile.png%2522%2520%250Aalt%253D%2522circular%2520x-large%2520avatar%2522%253E%250A%253Cimg%2520class%253D%2522circular-large-avatar%2522%2520src%253D%2522.%252Fassets%252Fprofile.png%2522%2520%250Aalt%253D%2522circular%2520large%2520avatar%2522%253E%250A%253Cimg%2520class%253D%2522circular-small-avatar%2522%2520src%253D%2522.%252Fassets%252Fprofile.png%2522%2520%250Aalt%253D%2522circular%2520small%2520avatar%2522%253E%250A%253Cimg%2520class%253D%2522circular-x-small-avatar%2522%2520src%253D%2522.%252Fassets%252Fprofile.png%2522%2520%250Aalt%253D%2522circular%2520x-small%2520avatar%2522%253E"
                 style="width: 70vw; height: 45vh; border:0; transform: scale(1); overflow:hidden;"
                 sandbox="allow-scripts allow-same-origin">
               </iframe>
             </section>
 
-            <h1 id="avatar-component-heading">Avatar</h1>
-
-                <h3>The Avatar component is used to represent a user, and displays the profile picture, initials or placeholder icon. It can be used in login page, notification bar, and in various parts of the project etc</h3>
-
+            
             
             <section class="component-section">
+                <h1 id="avatar-component-heading" class="component-headings">Avatar</h1>
```

Comment:
There should be only one `h1` tag per HTML page, read the heading hierarchy above.